recipes/rnabob/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: rnabob
+  version: "2.2.1"
+
+source:
+  url: http://eddylab.org/software/rnabob/rnabob.tar.gz
+  patches:
+    - manlocation.patch
+
+requirements:
+  build:
+
+  run:
+  
+test:
+  commands:
+    - rnabob -h
+
+about:
+  home: http://selab.janelia.org
+  license: University of Toronto
```

Comment:
http://conda.pydata.org/docs/building/meta-yaml.html#license-file
I guess you can remove it then.